plugins/modules/public_cloud_instance_starting.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: public_cloud_instance_starting
```

Comment:
Naming is hard, the module is not only managing start of instances. Maybe something like `public_cloud_instance_manage_state`